### DISCUSSION TITLE
Upgrade to PHP 7.4.11 breaks apache Catalina 10.5.7

### DISCUSSION THREAD

**Discussion by cuga-rajal** [Homebrew/discussions#93] [Everyday usage]
I'm using macOS built-in apache with homebrew's latest PHP. I've used this setup for many months without any issues. Upgrading from 7.4.10 to 7.4.11 has broken apache startup. There is nothing in the error-log to indicate the reason, even with logs set to debug mode. Running apachectl or httpd at the console does not give any useful messages. To resolve I have to downgrade all the way back to 7.3.x on Homebrew or use Apple's built-in PHP which is 7.3.11. I haven't been able to find a way to brew install a specific minor version such as 7.4.10.

I have a pretty generic setup with a few virtual hosts and self-signed ssl. The php config is the generic one as well as the httpd.conf. Problem happens on 2 different Mac Minis both running the latest version of Catalina.

I'm also using a recent mysql from the Oracle distro, relatively recent community release which hasn't changed across the time apache was/wasn't working. I doubt this is related since the problem happens at startup, and not while answering an http request. 

Any help appreciated.

**Comment by cuga-rajal** [comment]
When I install 7.4.11 with homebrew I can run php-v at the CLI and it shows me the correct version without error. But apache fails to start and so I cant see phpinfo(). Apachectl and httpd both fail with a meaningless console message.

**Reply by SMillerDev** [reply]
The homebrew php isn't meant to be used with the macOS apache (and I believe prints a message about it on install) if it worked before that was purely coincidence.

**Reply by cuga-rajal** [reply]
Thanks for the note -- It was working fine in the configuration for years. I confirmed the 7.4.10 to 7.4.11 also broke apache on Mojave.

Is there a debug method you can recommend so I can attempt to find the cause of the failure and hopefully fix.
